app/controllers/spree/admin/reports_controller_decorator.rb

```diff
@@ -244,7 +244,9 @@ def users_and_enterprises
   def xero_invoices
     if request.get?
       params[:q] ||= {}
-      params[:q][:completed_at_gt] = Time.zone.now.beginning_of_month
+      params[:q][:completed_at_gt] = Time.zone.today.beginning_of_month
+      params[:invoice_date] = Time.zone.today
+      params[:due_date] = Time.zone.today + 1.month
```

Comment:
Hey
I tried to extract the actual value of the defaults into constants in the class `XeroInvoicesReport`.
But then the test fails because of a missing dependency on ActiveSupport::TimeZone. I tried to `require 'active_support/time` before the class definition but anyway I need to give a value to Time.zone :/

Do you have an idea how It could be done nicely ?